registry/registry_db.go

```diff
@@ -676,7 +765,7 @@ func deleteNamespace(prefix string) error {
 }
 
 func getAllNamespaces() ([]*Namespace, error) {
-	query := `SELECT id, prefix, pubkey, identity, admin_metadata FROM namespace ORDER BY id ASC`
+	query := `SELECT id, prefix, pubkey, identity, admin_metadata, custom_fields FROM namespace ORDER BY id ASC`
```

Comment:
Will there be any backwards compatibility issues with the getAllNamespacesHandler returning namespaces with custom fields if called by something not expecting the custom fields?

Reply:
I can't think of the issue. If an older version of director/client is request all namespaces and we return extra fields they don't expect, then the fields won't be unmarshalled to their structs and no errors should be reported.